15_totalIntegers/solution/totalIntegers-solution.js

```diff
@@ -0,0 +1,15 @@
+const totalIntegers = function(obj, count = 0) {
+
+  if (typeof obj !== 'object' || obj === null) return;
+
+  const elements = Array.isArray(obj) ? obj : Object.values(obj);
+
+  for (const el of elements) {
+    if (Number.isInteger(el)) ++count;
+    if (typeof el === 'object' && el !== null) count += totalIntegers(el);
```

Comment:
```suggestion
    if (Number.isInteger(el)) {
      count++;
    } else if (typeof el === 'object' && el !== null) {
      count += totalIntegers(el);
    }
```
I believe post-fix increment is consistent with the curriculum's lessons and exercises.

`else if` would be more semantic here, even if `if` achieves the same result.
It also means the conditions can be chained.
Especially when chaining, having curly braces IMO is far more readable as the indentation makes it clearer to see what the code to run for each condition actually is.